src/commands/codepush/lib/environment.ts

```diff
@@ -0,0 +1,44 @@
+export interface EnvironmentInfo {
```

Comment:
These are going to be confusing with the EnvironmentInfo class already declared in the profile system. Wouldn't it be easier to integrate this with the existing profile/environment stuff?

Reply:
@christav The main idea was not to mix codepush configuration stuff with other AppCenter CLI configuration.
I mean here that original `EnvironmentInfo ` is different and also not sure if it will be good if we collide codepush with other configs. Also it seems that this solution (with using legacy service) could be not be needed in future and we will simply throw it away later at some point. 
May be we can just rename it e.g. to `CodePushEnvironmentInfo` and keep as it is now? Please let me know what do you think!

Reply:
I'd be happy with renaming it to `CodePushEnvironmentInfo`.

Reply:
fixed